src/components/Identity/processIdSyncs.js

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { assign, cookie, fireDestinations } from "../../utils";
+
+const millisecondsPerHour = 60 * 60 * 1000;
+
+const getControlObject = () => {
+  return JSON.parse(cookie.get("adobeIdSyncControl") || "{}");
```

Comment:
Let's make sure we use @jileeshs "alloy cookie" when it's finished (https://github.com/adobe/alloy/pull/9). Otherwise, we should use the namespace we're using elsewhere (https://github.com/adobe/alloy/blob/master/src/constants/namespace.js).